src/lib/event_mapper/glue_data_quality_event_mapper.py

```diff
@@ -52,8 +74,8 @@ def get_message_body(self):
             super().create_table_row(["State", self.get_resource_state()], style)
         )
 
-        context = self.event.get("detail", {}).get("context", {})
-        if context.get("contextType") == "GLUE_DATA_CATALOG":
+        context, context_type, run_id = self._get_context_and_run_id()
+        if context_type == "GLUE_DATA_CATALOG":
```

Comment:
maybe we can use context type constants declared in glue_manager here?

Reply:
sure, will update accordingly